components/Newsfeed/StyledNewsfeedComponents.tsx

```diff
@@ -11,9 +21,9 @@ export const Header = styled(Row)`
 export const HeaderTitle = styled(Col)`
   font-family: "Nunito";
   font-style: normal;
-  font-weight: 600;
-  font-size: 61px;
-  line-height: 125%;
+  font-weight: 700;
+  font-size: 40px;
```

Comment:
nit: if possible, would be nice to use one of our standard bootstrap sizes (and we should start pushing back on designs that deviate from those without due consideration). If not, don't sweat it for now.